src/components/service/dateFormatting.ts

```diff
@@ -0,0 +1,8 @@
+export default function formattingDate(date) {
```

Comment:
One more thing I missed here, `dateFormatting` is not component but is in `components` folder and it's not actually a service but a helper function. I suggest creating something like "utils" folder in the root and store it there